src/BSON/UTCDateTime.c

```diff
@@ -142,7 +143,7 @@ static bool php_phongo_utcdatetime_init_from_date(php_phongo_utcdatetime_t *inte
 
 	/* The following assignments use the same logic as date_format() in php_date.c */
 	sec = datetime_obj->time->sse;
-#if PHP_VERSION_ID >= 70200
+#if TIMELIB_VERSION >= 201705
```

Comment:
I did not use `TIMELIB_EXTENDED_VERSION` here, as that only seems to exist in PHP's master branch (added in https://github.com/php/php-src/commit/c0383b50399db7bd5996e87fed66be3fd7ded5d6 for what I assume is 7.3-dev).

Reply:
The ABI change was actually introduced in timelib version 2017.05beta5 (https://github.com/derickr/timelib/commit/f04e68166b2d1fcb8156ea9c5dd424308ae4f20f). Unfortunately, all of those beta releases of timelib define the same value for, which makes this unreliable to use for detection.